src/components/mobile-nav-item/__tests__/__snapshots__/mobile-nav-item.test.tsx.snap

```diff
@@ -0,0 +1,127 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MobileNavItem should render Expandable MobileNavItem 1`] = `
+<DocumentFragment>
+  <mock-styled.ul
```

Comment:
**suggestion:** I think we may want the parent element of a MobileNavItem to be an `<li>` element, right?

Reply:
>I think we may want the parent element of a MobileNavItem to be an <li> element, right?

thanks for pointing this out, I missed this crucial things really 🙂, because it's a list it should be rendered as `<li>`, and let's the consumer would declare the `<ul>` and `<nav>`